src/components/LaboratoryPageComponents/LaboratoryPageContent/hooks/useLaboratoryPageContent.ts

```diff
@@ -4,6 +4,7 @@ import { useNavigate } from 'react-router-dom';
 
 export const useLaboratoryPageContent = (id: string | undefined) => {
```

Comment:
Не забываем про ? для более простого создания типов
```suggestion
export const useLaboratoryPageContent = (id?: string) => {
```